t2

```diff
@@ -773,9 +773,9 @@ of files.
 	\*(DO esac
 	done
 .DE
-The \fB\(mic\fP flag is used in this command to
-force subsequent files to be created if they do not already exist.
-Otherwise, if the file does not exist, an error message is printed.
+By defult, if the file does not exist, it will be created.
+The \fB\(mic\fP flag is used to cause this command to
+print and error message rather than create a new file.
```

Comment:
Thanks for the contribution!

`default` seems to be misspelled. I believe you also meant `print an error message` rather than `and`.

Reply:
Done